flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/builtin/NtPidTable.java

```diff
@@ -0,0 +1,85 @@
+package com.flash3388.frc.nt.builtin;
+
+import com.beans.observables.ObservableDoubleValue;
+import com.flash3388.flashlib.robot.control.PidController;
+import com.flash3388.frc.nt.NtTable;
+import com.flash3388.frc.nt.beans.NtDoubleProperty;
+
+public class NtPidTable {
+    private static final String PID_TABLE_NAME = "Pid";
+    private static final String KP_ENTRY_NAME = "Kp";
+    private static final String KI_ENTRY_NAME = "Ki";
+    private static final String KD_ENTRY_NAME = "Kd";
+    private static final String KF_ENTRY_NAME = "Kf";
+    private static final String PID_INPUT_ENTRY_NAME = "Input";
+    private final NtTable mTable;
+
+    public NtPidTable(double kP, double kI, double kD, double kF, ObservableDoubleValue pidInput) {
```

Comment:
`PidInput` is officially called `process variable`, so maybe rename?

Reply:
ok

Reply:
It's still called `PidInput` in some place. It should be consistent.

Reply:
forgot about the methods sorry